src/common/utils/transactions.tsx

```diff
@@ -79,3 +76,83 @@ export function getTxTypeColor(txType: string) {
       return 'transactionTypes.tokenTransfer';
   }
 }
+
+export function getTxStatusIcon(tx: Transaction | MempoolTransaction) {
```

Comment:
thought for the future: If we switch these functions to static maps, we could more confidently exclude them from unit tests since they’d be pure 1:1 mappings with no logic. It’d also make them more ts friendly and more performant at runtime.